routes/api.php

```diff
@@ -533,20 +532,18 @@
 
       });
 
-
-
-
+    // pulling this out of resource route group to begin normalizing
+    Route::patch('/hardware/{asset}', [Api\AssetsController::class, 'update'])->name('api.assets.update');
```

Comment:
I think I just figured out why this is here - it's because we want to do the route-model binding and we need to change the route to respect that. Can we update the comment to make that clearer?

Reply:
Clarified a bit.